biz.aQute.resolve/build.gradle

```diff
@@ -0,0 +1,5 @@
+
+test {
+  systemProperty "org.slf4j.simpleLogger.defaultLogLevel", "Debug"
```

Comment:
Is this something we should commit to the repo? If so, why not for all test methods in the root build.gradle.

Finally, using `test` directly breaks the task configuration avoidance I added to the build.

Reply:
Will remove the file